### ISSUE TITLE
Ignore command line options are not exactly implemented as documented in README.md

### ISSUE THREAD

**Issue by ptrpfn** [ascmitc/mhl#55]
Minor documentation issue of the ignore funtionality:

The implementation:
```
 -i, --ignore TEXT               A single file pattern to ignore.
 --ignore_spec PATH              A file containing multiple file patterns to
                                 ignore.
```

The README.md:
```
$ ./ascmhl.py create [-i ignore pattern|-if /path/to/ignore-file.txt]
```

It needs to be decided what's better and change the README or command line option accordingly.

**Comment by jwaggs** [comment]
Agreed @ptrpfn - i could be convinced to go either way. My bad for not catching this.

Do you have a preference?

**Comment by ptrpfn** [comment]
I would say the most important thing is that we don't let the README divert too much from the actual implementation. The only thing I would find good is if you could offer a short-option for every option, just for consistency. If that short-option  where the ignore file is referenced then eventually becomes `-is`, `-if`, or something else is probably not that crucial, and probably best chosen by the implementor.

**Comment by jwaggs** [comment]
@ptrpfn I just opened PR #58 to address this issue. 

I went back and forth on the naming of the shorthand flags and decided to divert from your suggestion just a bit. Instead of `-is` and `-if` ... I chose `-i` and `-ii`. 

The reason for this is because both "is" and "if" make it sound like the flag includes matching items rather than excluding them. I feel `-i` for a single ignore item and `-ii` denoting a list of them in a file will be less confusing to end users.

The readme was also updated to match this change in the PR.

If you agree, go ahead and approve/merge at your leisure.
